omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -21,7 +21,7 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
      * @type {Function}
      */
     self.saveCondition = self.saveCondition || function () {
-        if($scope.condition.onSetDate > $scope.condition.endDate){
+        if($scope.condition.status == INACTIVE_STATUS && $scope.condition.onSetDate > $scope.condition.endDate){
```

Comment:
Is this validation supposed to be for only inactive conditions?

Reply:
Yes, because there is no end date for active conditions.

Reply:
Why don't you have the same check in the other pull request?

Reply:
I updated that PR.